### ISSUE TITLE
i请问一下lora fintune chatglm的时候，data_collector函数里面为什么没有attention_mask相关内容呀

### ISSUE THREAD

**Issue by zzy347964399** [liguodongiot/llm-action#8]

**Comment by zzy347964399** [comment]
GLM内部会对Input检查并加上，所以attention mask 和 position id 可以不丢给trainer #https://github.com/mymusise/ChatGLM-Tuning/issues/256
